code/modules/shuttle/arrivals.dm

```diff
@@ -173,4 +182,13 @@
 		stoplag()
 
 /obj/docking_port/mobile/arrivals/proc/QueueAnnounce(mob, rank)
-	LAZYADD(queued_announces, CALLBACK(GLOBAL_PROC, .proc/AnnounceArrival, mob, rank))
+	if(mode != SHUTTLE_CALL)
+		AnnounceArrival(mob, rank)
+	else
+		LAZYADD(queued_announces, CALLBACK(GLOBAL_PROC, .proc/AnnounceArrival, mob, rank))
+
+/obj/docking_port/mobile/arrivals/vv_edit_var(var_name, var_value)
+	if(var_name == "perma_docked")

```

Comment:
I don't think this is really needed, doubt anyone will be using VV to bypass feedback(and vv edits are logged and public anyway). Also, use to_chat() if you are going to keep this.